plugins/techdocs-backend/src/search/DefaultTechDocsCollator.ts

```diff
@@ -80,7 +84,9 @@ export class DefaultTechDocsCollator implements DocumentCollator {
       .map((entity: Entity) =>
         limit(async (): Promise<TechDocsDocument[]> => {
           const entityInfo = {
-            kind: entity.kind,
+            kind: this.legacyPathCasing
```

Comment:
I don't know how common it is to have cased namespaces or entity names, but it'd be good to keep the logic in sync w/the rest of TechDocs (lowercase all the types).